lib/exporters/jhipster_entity_exporter.js

```diff
@@ -124,7 +123,13 @@ function writeEntities(subFolder) {
   for (let i = 0, entityNames = Object.keys(configuration.entities); i < entityNames.length; i++) {
     const filePath = path.join(subFolder, toFilePath(entityNames[i]));
     const entity = updateEntityToGenerateWithExistingOne(filePath, configuration.entities[entityNames[i]]);
-    fs.writeFileSync(filePath, JSON.stringify(entity, null, 2).concat('\n'));
+    if (configuration.returnMergedEntity) {
```

Comment:
Shouldn't it always be true? I mean, I don't see cases where we don't want that...

Reply:
Your call.
I did for backward compatibility.
But since this can be considered a bug, a minor release should be enough for this change.
The only use case I can think is if some blueprint wants to change the behavior. But I don’t think we should support that.

Reply:
Ah! good point. You're absolutely right! Let's make it the default, we'll do release including this later (I still have to finish the "options in app" thing)